tweakwcs/linalg.py

```diff
@@ -56,7 +56,7 @@ def _is_longdouble_lte_flt_type(flt_type=np.double):
 
 def _find_max_linalg_type():
     max_type = None
-    for np_type in np.sctypes['float'][::-1]:  # pragma: no branch
+    for np_type in np.core.sctypes['float'][::-1]:  # pragma: no branch
```

Comment:
While looking into the `np.sctypes` removal I found the  following comment mentioning that `numpy.core` is likely to also be removed in numpy 2.0:
https://github.com/numpy/numpy/pull/24376#issuecomment-1695909212

Is the goal of this for loop to find the largest float that can survive line 61 without a `TypeError`? If so, perhaps we explicitly list the types as the following:
```
for np_type in [np.longdouble, np.float64, np.float32, np.float16]:
```
As discussed in [the numpy docs](https://numpy.org/doc/stable/user/basics.types.html#extended-precision) `longdouble` will be 80 bits on systems that support it (like my linux laptop) and 64 on systems that don't (like my M1 mac) (also note the description of how `float128` and `float96` are only 80 bits with padding). Also, does the 'fall back' non-numpy method offer 64 bit precision? If so, should we leave off some of the lower precision options (`float16` and `float32`)?

Reply:
>Also, does the 'fall back' non-numpy method offer 64 bit precision?https://docs.python.org/3/tutorial/floatingpoint.html#representation-error

Reply:
Thanks for detective work! I am disappointed with `numpy` decision to remove `sctypes`.

Reply:
```suggestion
    for np_type in [np.longdouble, np.float64, np.float32]:  # pragma: no branch
```
Thanks! So something like the above looks good to you?